src/Api/Vault/AuthorizationHandlers/Collections/BulkCollectionAuthorizationHandler.cs

```diff
@@ -191,10 +191,9 @@ private async Task<bool> CanUpdateCollectionAsync(ICollection<Collection> resour
             return true;
         }
 
-        // If V1 is enabled, Owners and Admins can update any collection only if permitted by collection management settings
+        // Owners and Admins can update any collection only if permitted by collection management settings
         var organizationAbility = await GetOrganizationAbilityAsync(org);
-        var allowAdminAccessToAllCollectionItems = !_featureService.IsEnabled(FeatureFlagKeys.FlexibleCollectionsV1) ||
-                                                   organizationAbility is { AllowAdminAccessToAllCollectionItems: true };
+        var allowAdminAccessToAllCollectionItems = organizationAbility is { AllowAdminAccessToAllCollectionItems: true };
         if (allowAdminAccessToAllCollectionItems && org is { Type: OrganizationUserType.Owner or OrganizationUserType.Admin })
```

Comment:
Please update to use the `AllowAdminAccessToAllCollectionItems` private method. (We introduced it partway through, so it's not used consistently.)

e.g.

```cs
if (await AllowAdminAccessToAllCollectionItems(org) &&
  org is { Type: OrganizationUserType.Owner or OrganizationUserType.Admin })
```